src/services/dispatchers/dispatchers/index.ts

```diff
@@ -40,6 +41,14 @@ export default (
       return await query(options);
     },
 
+    async findAll(this: DispatchersService, options) {
+      return findAllGeneric<Dispatcher>(this.find, options);
```

Comment:
I think both instances of `this.find` should be `query` 🤔

Reply:
Copied from: https://github.com/ExtraHorizon/javascript-sdk/blob/358ec06cb9aad18774d7dfac1f607c6d399fa948/src/services/tasks/tasks.ts but there we don't have query, so I will change it 😄